java-client/build.gradle.kts

```
@@ -205,23 +205,24 @@ dependencies {
 
     // EPL-2.0 OR BSD-3-Clause
     // https://eclipse-ee4j.github.io/yasson/
-    testImplementation("org.eclipse", "yasson", "2.0.2") {
+    testImplementation("org.eclipse", "yasson", "2.0.4") {
         // Exclude Glassfish as we use Parsson (basically Glassfish renamed in the Jakarta namespace).
         exclude(group = "org.glassfish", module = "jakarta.json")
     }
 
-    // EPL-1.0
-    // https://junit.org/junit4/
-    testImplementation("junit", "junit" , "4.13.2")
+    // EPL-2.0
+    // https://junit.org/junit5/
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.8.2")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.8.2")
 
     // MIT
     // https://github.com/classgraph/classgraph
-    testImplementation("io.github.classgraph:classgraph:4.8.116")
+    testImplementation("io.github.classgraph:classgraph:4.8.146")
 
     // MIT
     // https://www.testcontainers.org/
-    testImplementation("org.testcontainers", "testcontainers", "1.15.3")
-    testImplementation("org.testcontainers", "elasticsearch", "1.15.3")
+    testImplementation("org.testcontainers", "testcontainers", "1.17.1")
+    testImplementation("org.testcontainers", "elasticsearch", "1.17.1")
```

A new version of TestContainers has been released:

```suggestion
    testImplementation("org.testcontainers", "elasticsearch", "1.17.2")
```